perception/autoware_map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1149,11 +1161,29 @@ void MapBasedPredictionNode::objectsCallback(const TrackedObjects::ConstSharedPt
         std::vector<PredictedPath> predicted_paths;
         double min_avg_curvature = std::numeric_limits<double>::max();
         PredictedPath path_with_smallest_avg_curvature;
-
         for (const auto & ref_path : ref_paths) {
-          PredictedPath predicted_path = path_generator_->generatePathForOnLaneVehicle(
-            yaw_fixed_transformed_object, ref_path.path, prediction_time_horizon_.vehicle,
-            lateral_control_time_horizon_, ref_path.speed_limit);
+          PredictedPath predicted_path{};
+          // convert PredictedRefPath to PredictedPath
+          predicted_path.path = ref_path.path;
+          predicted_path.confidence = ref_path.probability;
+
+          if (current_lanelets.front().is_bidirectional) {
+            if (isLeft(transformed_object)) {
+              predicted_path = path_generator_->generateShiftedPathForOnLaneVehicle(
+                yaw_fixed_transformed_object, predicted_path, prediction_time_horizon_.vehicle,
+                lateral_control_time_horizon_, bound_to_centerline / 2, ref_path.speed_limit);
```

Comment:
[NITS] In terms of the performance, multiplication is better than division.
```suggestion
                lateral_control_time_horizon_, 0.5 * bound_to_centerline, ref_path.speed_limit);
```